ci/jenkins/test-vmc.sh

```diff
@@ -353,6 +353,14 @@ function deliver_antrea {
     # Clean up dangling images generated in previous builds. Recent ones must be excluded
     # because they might be being used in other builds running simultaneously.
     docker image prune -f --filter "until=1h" > /dev/null
+    free_space=$(df -h / | awk 'NR==2 {print $4}' | sed 's/G//')
+    free_space_threshold=30
+    if [[ $free_space -lt $free_space_threshold ]]; then
+        # If cleaning up dangling images unused in the last hour doesn't free up sufficient disk space,
+        # we will have to clean up all dangling images and antrea related images to release enough disk space.
+        docker images | grep 'antrea' | awk '{print $3}' | xargs -r docker rmi || true
+        docker images | grep '<none>' | awk '{print $3}' | xargs -r docker rmi || true
```

Comment:
It could be the case where in the builds might be used at other builds running simulataneously , since the pruning is done for images until 1 hour, idea is to allow image to be used if it was created within an hour and escape the prune, what if we use -af in docker prune and delete all unused to free more space ?

Reply:
Yes, `docker image prune -af` can remove all unused image including dangling images.